tests-configuration-infinispan/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/cache/container/scattered/cache/store/jdbc/WriteBehaviourTest.java

```diff
@@ -100,6 +101,7 @@ public void change2ModificationQueueSize() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
What is the reason for ignore?

Reply:
UI model change, attribute that is subject of testing became deprecated. Resolving.